final_prep/annotations.rb

```diff
@@ -4,40 +4,57 @@
 
 # Build a Bear
 
+# declare build_a_bear object with name, age, fur, clothes, and special_power as attributes
```

Comment:
This is actually declaring a `build_a_bear` function that will return an object/hash.